utils/chunked_vector.hh

```diff
@@ -106,6 +106,7 @@ public:
     template <typename Iterator>
     chunked_vector(Iterator begin, Iterator end);
     chunked_vector(std::initializer_list<T> x);
+    chunked_vector(const T& v) : chunked_vector(1, v) {}
```

Comment:
I'm not sure this is a good idea. First, std::vector doesn't have such a constructor.  Second, it adds ambiguity in the case of `chunked_vector<int>` - does `chunked_vector<int>(3)` mean a vector of three zeros (this is the case today) or a vector of one 3?

With the new initializer list constructor, wouldn't `chunked_vector<int>({3})` work?

Reply:
Good point. Using an initializer list to construct a single element is the way to go.
And the interface should definitely be compatible with std::vector.